app/assets/stylesheets/common/_core.scss

```diff
@@ -237,18 +237,43 @@ label.required::after, th.required::after {
 .ReactTable {
   .rt-thead,
   .rt-tbody {
+    border-right: 1px solid $line;
+    border-bottom: 1px solid $line;
     overflow-x: hidden;
     overflow-y: scroll;
   }
 
   .rt-thead .rt-hidden,
-  .rt-hidden {
+  .rt-hidden,
+  .rt-thead .rt-td {
+    border-right: 1px solid $line;
+    border-bottom: 1px solid $line;
     padding: 0;
   }
 
+  .rt-thead.-filters .rt-th {
+    border-right: 1px solid $line;
+    border-bottom: 1px solid $line;
+  }
+
+  .rt-tbody .rt-td {
+    border-right: 1px solid $line;
+    border-bottom: 1px solid $line;
+  }
+
+  .rt-th.rt-resizable-header-content.-cursor-pointer {
+    border: 1px solid $line;
+  }
+
+  .rt-thead .rt-th, .rt-thead .rt-td {
+    color: $line;
+  }
+
   .rt-td:last-child {
     // Hack to ensure scrollbar doesn't cover cell contents.
     padding-right: 22px;
+    border-right: 1px solid $line;
```

Comment:
I'm not sure why these two rules are necessary, since this should just apply to the last cell.